images/image.mk

```diff
@@ -18,7 +18,10 @@
 override GOOS=linux
 
 ifeq ($(origin DOCKERCMD),undefined)
-DOCKERCMD := docker
+	DOCKERCMD?=$(shell docker version >/dev/null 2>&1 && echo docker)
+	ifeq ($(DOCKERCMD),)
+                DOCKERCMD=$(shell podman version >/dev/null 2>&1 && echo podman)
```

Comment:
It looks like this indent is not correct.

Reply:
it's not clear to me what the indent should be for this - just removed all indention.